client/components/layout/styles.js

```diff
@@ -67,14 +67,53 @@ export const Navbar = styled.nav`
 export const NavbarLink = styled.a`
   font-size: 16px;
   line-height: 24px;
-  color: ${props => props.theme.colors.light};
+  color: ${props => props.theme.colors.textMenu};
   margin-right: 32px;
 
   &:last-of-type {
     margin-right: 0;
   }
 `;
 
+export const NavbarToggleSwitch = styled.label`
+  position: relative;
+  display: inline-block;
+  width: 46px;
+  height: 26px;
+  margin-left: 32px;
+  margin-right: 4px;
+`;
+
+export const ToggleSwitchSlider = styled.span`
+  position: absolute;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background: ${props => props.theme.colors.backgroundPrimary};
+  border-radius: 26px;
```

Comment:
The toggle looks great, except when in dark mode you can't see the container the toggle switch is in so it looks just like a green circle sitting in the nav 😅.

Maybe something like this on line 96 to be able to see the switch background:
`${props => props.isDarkTheme && 'filter: contrast(1.3)'};`
I tried it on my local and it makes the switch BG a little darker and the the toggle looks more like a toggle.

Reply:
@ZumDeWald Thanks for your feedback! Whoops, I forgot to remove those files. (I was originally implemented something with sun/moon images, but I changed it to the toggle switch).

Oohh, that's a nice approach to create a constrasting background without having to create additional color variables. I'll add it to the PR. Thanks :)